testonedrivesdk/test_models.py

```diff
@@ -29,5 +29,35 @@ def test_serialization(self):
         assert item.last_modified_date_time.isoformat()+"Z" == response["lastModifiedDateTime"]
 
 
+    def test_serialization_different_datetime(self):
+        """
+        Test the serialization of the dict-backed models, seeing that
+        the correct objects are returned when called. Specifically,
+        ensure that the datetime can be parsed correctly when the format
+        does not fit exactly what we always return
+        """
+        ref = ItemReference();
+        ref._prop_dict = {"id": "thisisa!test"}
+
+        response = {"name":"test1", "folder":{}, "parentReference":ref._prop_dict, "lastModifiedDateTime": "2015-07-09T22:22:53.99Z"}
+
+        item = Item();
+        item._prop_dict = response
+
+        assert type(item.folder) is Folder
+        assert item.name == "test1"
+        assert type(item.parent_reference) is ItemReference
+        assert item.parent_reference.id == "thisisa!test"
```

Comment:
make this its own variable at the top instead of embedding the literal everywhere